stack/setup/main.tf

```
@@ -0,0 +1,50 @@
+locals {
```

what do you think about adding EIPs here? We'll need an EIP for the NAT (see https://stackoverflow.com/questions/61265108/aws-ecs-fargate-resourceinitializationerror-unable-to-pull-secrets-or-registry)

naming schema could be:

```
"${local.project}-${eip_name}"
```

and we create 1 EIP per `eip_name` given (as a list).
we'd also have to set `vpc` to true/false (true by default?)


```terraform
resource "aws_eip" "main" {
count = 3
```suggestion
locals {
```
  vpc = var.vpc
  tags = {
    Name        = "${local.project}-${eip_name}"
    Project     = local.project
    Environment = "production"
  }
}
```